src/main/java/uk/gov/digital/ho/pttg/application/HmrcHateoasClient.java

```diff
@@ -218,7 +218,13 @@ private String stripPlaceholderQueryParams(String href) {
 
             try {
                 final Resource<String> matchedIndividual = performMatchedIndividualRequest(matchUrl, accessToken, candidateNames.get(retries), individual.getNino(), individual.getDateOfBirth());
-                log.info("Successfully matched individual {}", individual.getNino(), value(EVENT, HMRC_MATCHING_SUCCESS_RECEIVED));
+
+                log.info("Successfully matched individual {}",
+                        individual.getNino(),
+                        value("combination", String.format("%d of %d", retries + 1, candidateNames.size())),
+                        value("name-matching-analysis", candidateNames.get(retries).derivation()),
+                        value(EVENT, HMRC_MATCHING_SUCCESS_RECEIVED));
+
```

Comment:
Is there some way we can log out the input names analysis in the case that we never get a match?  Probably requires a new ticket...

Reply:
I've created this JIRA: https://jira.ipttools.info/browse/EE-11594